gems/sorbet-runtime/lib/types/private/methods/_methods.rb

```diff
@@ -10,7 +10,15 @@ module T::Private::Methods
   ARG_NOT_PROVIDED = Object.new
   PROC_TYPE = Object.new
 
-  DeclarationBlock = Struct.new(:mod, :loc, :blk)
+  class DeclarationBlock
+    attr_accessor :mod, :loc, :blk
+
+    def initialize(mod, loc, blk)
+      @mod = mod
+      @loc = loc
+      @blk = blk
+    end
```

Comment:
Is this not equivalent to `Struct.new`?

Reply:
It's supposed to be, to start, but then I add things related to 'final' to it.

I wanted to store the bit that says whether this sig is a final sig or not with the rest of the relevant data, and this seemed like a way to do it. Unsure if it's the best way.